opengrok-indexer/src/main/java/org/opengrok/indexer/history/History.java

```diff
@@ -107,24 +108,19 @@ public boolean hasFileList() {
      * tag list, {@code false} otherwise
      */
     public boolean hasTags() {
-        // TODO Use a private variable instead of for loop?
```

Comment:
So far this is only used once in `history.jsp` so I do not think memoizing or precomputing the result in the constructor is justified.